repo2docker/buildpacks/conda/activate-conda.sh

```diff
@@ -1,14 +1,14 @@
 # enable conda and activate the notebook environment
-eval $(micromamba shell hook -s posix -p ${CONDA_DIR})
+eval $(micromamba shell hook -s posix -r ${CONDA_DIR})
 for name in conda mamba; do
```

Comment:
Ok let's remove `mamba` here and just leave `conda` for a first test (only with `micromamba`)